src/components/ActivitiesList/mocks.ts

```diff
@@ -3,128 +3,170 @@ import { Activity } from "@components/ActivitiesList/ActivitiesList";
 export const ACTIVITIES_MOCK: Activity[] = [
   {
     id: 1,
+    name: "Death on the Nile",
+    description:
+      "Watch a movie starring Kenneth Branaugh about solving a murder mystery.",
+    voteCountUp: 2,
+    voteCountDown: 0,
+    memberCount: 2,
+    category: "Indoor",
+    location: "My place",
+    date: "2023-05-11",
+    timeStart: "7:30 PM",
+    timeEnd: "10:00 PM",
+  },
+  {
+    id: 2,
+    name: "Make Pasta",
+    description:
+      "Make pasta from scratch using flour and egg. Then cook and use favorite sauce.",
+    voteCountUp: 2,
+    voteCountDown: 1,
+    memberCount: 2,
+    category: "Indoor",
+    location: "My place",
+    date: "2023-05-11",
+    timeStart: "7:30 PM",
+    timeEnd: "10:00 PM",
+  },
+  {
+    id: 3,
+    name: "Swim 10 laps",
+    description: "Swim 10 laps in the pool.",
+    voteCountUp: 0,
+    voteCountDown: 1,
+    memberCount: 2,
+    category: "Indoor/Outdoor",
+    location: "My place",
+    date: "2023-05-11",
+    timeStart: "7:30 PM",
+    timeEnd: "10:00 PM",
+  },
+  {
+    id: 4,
+    name: "Wedding Crasher",
+    description: "Go to a random wedding and crash",
+    voteCountUp: 4,
+    voteCountDown: 1,
+    memberCount: 2,
+    category: "Indoor",
+    location: "My place",
+    date: "2023-05-11",
+    timeStart: "7:30 PM",
+    timeEnd: "10:00 PM",
+  },
+  {
+    id: 5,
+    name: "Netflix and Relax",
+    description:
+      "Pick a movie on Netflix and relax. No discration. You can also discuss the movie like pretentious intellectuals.",
+    voteCountUp: 1,
+    voteCountDown: 1,
+    memberCount: 2,
+    category: "Indoor",
+    location: "My place",
+    date: "2023-05-11",
+    timeStart: "7:30 PM",
+    timeEnd: "10:00 PM",
+  },
+  {
+    id: 6,
+    name: "Play musical instruments",
+    description: "Learn an instrument of choice and jam together.",
+    voteCountUp: 3,
+    voteCountDown: 1,
+    memberCount: 2,
+    category: "Indoor",
+    location: "My place",
+    date: "2023-05-11",
+    timeStart: "7:30 PM",
+    timeEnd: "10:00 PM",
+  },
+  {
+    id: 7,
+    name: "Batting Cage Therapy",
+    description: "Go to a batting cage and hit your problems away.",
+    voteCountUp: 2,
+    voteCountDown: 1,
+    memberCount: 2,
+    category: "Outdoor",
+    location: "My place",
+    date: "2023-05-11",
+    timeStart: "7:30 PM",
+    timeEnd: "10:00 PM",
+  },
+  {
+    id: 8,
     name: "Kayaking",
     description: "Relax on a nice sunny day with a trip down the river.",
     voteCount: 4,
     category: "outdoor",
   },
 
   {
-    id: 2,
+    id: 9,
     name: "Movies",
     description:
       "If the weather is bad outside, enjoy a cozy movie in a movie theater.",
     voteCount: 2,
```

Comment:
let's make sure the mock data from the other devs' PRs match the same structure in terms of how their votes are being tracked